File: java programs/23_program.java
import java.util.*;
class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    double r = sc.nextDouble();
    double pi = 3.141592653589793;
    double area = pi * r * r;
    double circumference = 2 * pi * r;
    System.out.println(area + " " + circumference);
  }
}

File: c programs/23_program.c
#include <stdio.h>
#define PI 3.141592653589793
int main() {
    double r, area, circumference;
    scanf("%lf", &r);
    area = PI * r * r;
    circumference = 2 * PI * r;
    printf("%.6lf %.6lf", area, circumference);
    return 0;
}
s
